packages/storybook-addon-baseline-grid/package.json

```diff
@@ -0,0 +1,106 @@
+{
```

Comment:
A few things I'm seeing missing from this package file:
- LGPL3 license instead of MIT
- Links to our Github repo & issues page
- Biome check scripts
- Dependency on our biome config & typescript config

Reply:
> * LGPL3 license instead of MIT
> * Links to our Github repo & issues page

Will do those as part of this PR

> * Biome check scripts
> * Dependency on our biome config & typescript config
> * Remove Prettier dependency and `.prettierrc` and `.prettierignore` as we are using Biome, not Prettier

This is covered by a follow up #90